src/view_text.cpp

```diff
@@ -447,14 +447,15 @@ void View::DrawRend(DeviceContext *dc, Rend *rend, TextDrawingParams &params)
         dc->GetFont()->SetPointSize(dc->GetFont()->GetPointSize() / SUPER_SCRIPT_FACTOR);
     }
 
-    // Do not render the box or circle if the content is empty
-    if (rend->HasContentBB()) {
-        if (rend->HasRend()) {
-            params.m_enclosedRend.push_back(rend);
-            params.m_x = rend->GetContentRight() + m_doc->GetDrawingUnit(100);
-            params.m_explicitPosition = true;
-            params.m_enclose = rend->GetRend();
-        }
+    const bool isEnclosed = (rend->HasRend())
+        ? (((rend->GetRend() - TEXTRENDITION_box) | (TEXTRENDITION_tbox - rend->GetRend())) >= 0)
+        : false;
```

Comment:
This is cryptic. What's the logic behind it

Reply:
Since we are talking about four values I think the simplest would be to have a helper function (on `Rend`?) to check it.